packages/customize-widgets/src/components/sidebar-block-editor/sidebar-adapter.js

```diff
@@ -173,6 +173,8 @@ export default class SidebarAdapter {
 			number = widgetModel.get( 'multi_number' );
 		}
 
+		widget.instance.is_widget_customizer_js_value = true;
```

Comment:
The mutation here is confusing me. How does the server end up receiving this? I don't see `widget` referenced after this line in this function.

Reply:
There's one when setting the widget's instance via the JS API. So that whenever it's saved to the server, it will carry the value to it.

```js
setting.set( widget.instance );
```